internal/provider/resource_consul_cluster_test.go

```diff
@@ -38,24 +41,24 @@ func setTestAccConsulClusterConfig(consulCluster string) string {
 
 	data "hcp_consul_versions" "test" {}
 
-	%s
+	%[1]s
 	
-	data "hcp_consul_cluster" "test" {
-		cluster_id = hcp_consul_cluster.test.cluster_id
+	data "hcp_consul_cluster" "%[2]s" {
```

Comment:
The resource name should stay the same.

```suggestion
	data "hcp_consul_cluster" "test" {
```